xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -47,10 +47,19 @@
         <Parameter Name="window" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="quality">To be added.</param>
-        <param name="window">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="quality">A number representing quality of the Brotli compression. 0 is the minimum (no compression), 11 is the maximum.</param>
+        <param name="window">A number representing the encoder window bits. The minimum value is 10, and the maximum value is 24.</param>
+        <summary>Initializes an instance of <see cref="T:System.IO.Compression.BrotliEncoder" /> class using the specified quality and window.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="quality" /> is not between the minimum value of 0 and the maximum value of 11.
+
+-or-
+
+<paramref name="window" /> is not between the minimum value of 10 and the maximum value of 24.
+        </exception>
+        <exception cref="T:System.InvalidOperationException">Quality and window cannot be changed at the current encoder state.</exception>
```

Comment:
This and the next exception are surprising in a constructor. Instantiation should initialize the encoder state. And there shouldn't be a call to Dispose before the instantiation.

Reply:
You're right, this is interesting. I followed the constructor code and reached these two points, but I am not sure if when creating a new instance we would ever reach those exceptions. Let me review this code again, see if there's a chance of hitting the two exceptions. But for now, I'll remove them:
```suggestion
```